sources/packages/api/src-sql/Restrictions/Add-col-restriction-category.sql

```diff
@@ -0,0 +1,13 @@
+-- Add restriction_category to restrictions table.
+ALTER TABLE
+    sims.restrictions
+ADD
+    COLUMN IF NOT EXISTS restriction_category VARCHAR(50) NOT NULL DEFAULT 'Federal';
```

Comment:
I think the type of the columns should be `restriction_types`
![image](https://user-images.githubusercontent.com/77353155/148125787-bec7435f-c208-4a53-844e-7c00994ffa30.png)